test/end-to-end/shared/GridUtils.js

```diff
@@ -29,6 +29,13 @@ function getCell(gridId, row, col) {
     .element(by.repeater('(colRenderIndex, col) in colContainer.renderedColumns track by col.uid').row(col));
 }
 
+function getCellName(gridId, row, col) {
```

Comment:
it's not possible to delete old function getCell, because 35 test integration track column container by col.uid and not by col.colDef.name, it for that i have add a new function getCellName who track by col.colDef.name